src/main/bash/envs/kobman-Aries-dev.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+function __kobman_install_Aries-dev
```

Comment:
@asa1997 is the A cap consistent with all the environment names, let us have all function names small caps

Reply:
kob install -env Aries-dev
- We have to follow the name of the env given in the installation command as such.
- Aries-dev from install command is stored to a vairable and using this variable install function of the environment is called.
 $environment=Aries-dev
 __kobman_install_$environment "environment"
 - The first $environment is used to identify the installation function of the environment.